### PR TITLE
[FEAT] Incubator - Community Garden

### PR DESCRIPTION
**Pull request by tiffanydys**
# Description
Tadpole incubator, incubate your tadpole + frog for whitelist token rewards 😄

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
`yarn dev`

Screen Shot 2022-10-27 at 9 49 15 PM
Screen Shot 2022-10-27 at 11 07 40 PM
Screen Shot 2022-11-01 at 4 59 04 AM

`yarn test`
Screen Shot 2022-11-01 at 5 35 08 AM

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/community/scientist/ScientistModal.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  const [machine, send] = useMachine(incubateMachine);
+  const { state, errorCode } = machine.context;
+
+  const handleClose = () => onClose();
+
+  const [tab, setTab] = useState<"inventory" | "incubator">("inventory");
+
+  return (
+    <Modal centered show={isOpen} onHide={handleClose}>
+      <Panel className="relative max-w-5xl">
```
Comment (adamhannigan) [inline src/features/community/scientist/ScientistModal.tsx:38]: Screen Shot 2022-11-03 at 2 04 30 pm

For the 'tab' style you will need to put some padding at the top of your Panel. See attached for how tabs should appear.

**Review by adamhannigan — APPROVED** [review]
Looks good to me. Just a small UI inconsistency with the Tab layout that we will need to tidy up before merging in 👍

**Comment by tiffanydys** [comment]
> Looks good to me. Just a small UI inconsistency with the Tab layout that we will need to tidy up before merging in 👍 

Hi @adamhannigan, the UI changes are done and I've also added in the verified contracts deployed on mainnet. ❤️

**Comment by adamhannigan** [comment]
@tiffanydys Looks good to me, just a small CI error at the moment

**Comment by tiffanydys** [comment]
> @tiffanydys Looks good to me, just a small CI error at the moment

Pulled from main. Thanks @adamhannigan 😊

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
It looks like when you updated to main there are a couple of import errors. I think the `nfts` folder was renamed to `sfts` in `assets`. Can you please update the imports?

**Comment by tiffanydys** [comment]
> It looks like when you updated to main there are a couple of import errors. I think the `nfts` folder was renamed to `sfts` in `assets`. Can you please update the imports?

Edited the imports to one file and it seems to be okay now. Thanks!

**Pull request merged**